cmd/manager/main.go

```diff
@@ -66,6 +69,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	// we have two namespaces that we need to watch
+	// 1. tuned namespace
+	// 2. None namespace
+	namespaces := []string{
+		components.NamespaceNodeTuningOperator,
+		metav1.NamespaceNone,
```

Comment:
Could you please add a one-line comment to remind our future selves what "None namespace" means in this context? (e.g. watch all the namespaces, even though IIRC this is not the case)

Reply:
done